packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/CamConfiguration.swift

```diff
@@ -0,0 +1,87 @@
+// Copyright 2013 The Flutter Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import AVFoundation
+import CoreMedia
+import Foundation
+import UIKit
+
+// Import Objective-C part of the implementation when SwiftPM is used.
+#if canImport(camera_avfoundation_objc)
+  import camera_avfoundation_objc
+#endif
+
+/// Factory block returning an FLTCaptureDevice.
+/// Used in tests to inject a device into DefaultCamera.
+typealias CaptureDeviceFactory = (String) -> FLTCaptureDevice
+
+typealias AudioCaptureDeviceFactory = () -> FLTCaptureDevice
+
+typealias CaptureSessionFactory = () -> FLTCaptureSession
+
+typealias AssetWriterFactory = (URL, AVFileType, inout NSError?) -> FLTAssetWriter?
```

Comment:
can you add label so it's more readable? You may have to put an underscore there `_ assetURL: URL`